### DISCUSSION TITLE
Welcome to OctoprintKlipperPlugin Discussions!

### DISCUSSION THREAD

**Discussion by thelastWallE** [thelastWallE/OctoprintKlipperPlugin#56] [Announcements]
## 👋 Welcome!
I activated discussions on the repository to get better in contact with you all.

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Jeromehouseboat** [comment]
Wanted to say thank you so much for your time on this! Much improved klipper interface over stock octoprint. 

I think the to-dos in the 0.4.0 will be helpful. I’m curious if you wouldn’t mind sharing your macro list from you picture example? Obviously would require editing, but that would be extremely beneficial for me, as I’m having to rebuild from scratch since my last Pi went down.
